kernel/src/actions/mod.rs

```diff
@@ -90,7 +91,160 @@ impl Metadata {
     }
 }
 
-#[derive(Default, Debug, Clone, PartialEq, Eq, Schema)]
+/// Features table readers can support as well as let users know
```

Comment:
can we move these into a `features` mod? In general I think it would be best to keep just the actual "Actions" in this mod (although it's probably already not perfect on that front)

Reply:
done - i took the liberty of moving the `column_mapping` module as well into a `features` module.